.github/workflows/conda-ci.yml

```diff
@@ -0,0 +1,128 @@
+  
+jobs:
+  build-with-conda-dependencies:    
+    name: '[conda:${{ matrix.os }}]'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        build_type: [Release]
+        os: [ubuntu-20.04, macos-10.15, windows-2019]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-update-conda: true
+
+    - name: Install files to enable compilation of mex files [Conda/Linux]
+      if: contains(matrix.os, 'ubuntu') 
+      run: |
+        curl -L -O https://github.com/robotology/robotology-vcpkg-ports/releases/download/storage/msdk_R2020b_mexa64.zip
+        unzip msdk_R2020b_mexa64.zip
+        rm msdk_R2020b_mexa64.zip
+        echo "GHA_Matlab_ROOT_DIR=${GITHUB_WORKSPACE}/msdk_R2020b_mexa64" >> $GITHUB_ENV
+        echo "GHA_Matlab_MEX_EXTENSION=mexa64" >> $GITHUB_ENV
+                
+    - name: Install files to enable compilation of mex files [Conda/macOS]
+      if: contains(matrix.os, 'macos') 
+      run: |
+        curl -L -O https://github.com/robotology/robotology-vcpkg-ports/releases/download/storage/msdk_R2020a_mexmaci64.zip
+        unzip msdk_R2020a_mexmaci64.zip
+        rm msdk_R2020a_mexmaci64.zip
+        echo "GHA_Matlab_ROOT_DIR=${GITHUB_WORKSPACE}/msdk_R2020a_mexmaci64" >> $GITHUB_ENV
+        echo "GHA_Matlab_MEX_EXTENSION=mexmaci64" >> $GITHUB_ENV
+        
+    - name: Install files to enable compilation of mex files [Conda/Windows]
+      if: contains(matrix.os, 'windows') 
+      shell: bash
+      run: |
+        curl -L -O https://github.com/robotology/robotology-vcpkg-ports/releases/download/storage/msdk_R2020a_mexw64.zip
+        unzip msdk_R2020a_mexw64.zip
+        rm msdk_R2020a_mexw64.zip
+        echo "GHA_Matlab_ROOT_DIR=${GITHUB_WORKSPACE}/msdk_R2020a_mexw64" >> $GITHUB_ENV
+        echo "GHA_Matlab_MEX_EXTENSION=mexw64" >> $GITHUB_ENV
+
+    - name: Dependencies [Conda]
+      shell: bash -l {0}
+      run: |
+        # Install mamba (https://github.com/mamba-org/mamba) as first step 
+        conda install -c conda-forge mamba
+        # Compilation related dependencies 
+        mamba install -c conda-forge cmake compilers make ninja pkg-config
+        # YARP dependencies 
+        mamba install -c conda-forge ace eigen ycm-cmake-modules
+        # Uncomment when we are compatible with releases in conda yarp
+        # mamba install -c conda-forge -c robotology yarp 
+
+
+    - name: Install YARP [Conda/Linux&macOS]
+      if: contains(matrix.os, 'macos') || contains(matrix.os, 'ubuntu')
+      shell: bash -l {0}
+      run: |
+        git clone https://github.com/robotology/yarp
+        cd yarp
+        mkdir build
+        cd build
+        cmake -GNinja -DCMAKE_INSTALL_PREFIX=${CONDA_PREFIX} -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DYARP_COMPILE_EXECUTABLES:BOOL=OFF ..
+        cmake --build . --config ${{ matrix.build_type }}        
+        cmake --install . --config ${{ matrix.build_type }}
```

Comment:
This will be removed when line 66 will be enabled ?

Reply:
Yes. Actually it will depend, as the `master` branch may need to have specific features that are not released.